### DISCUSSION TITLE
Knowledge base

### DISCUSSION THREAD

**Discussion by mirmoh3n** [LiveHelperChat/livehelperchat#2145] [General]
Hello. Is it possible to add a knowledge base feature like other existing online chat systems? The graphic of the chat window of similar systems looks much better.  The system is similar https://www.tawk.to

**Comment by remdex** [comment]
Few points
 * It's not much better it's different.
 * As for knowledge base you should atelast invest some time and paste sketch, screenshot or whatever for sample.

As for knowledge base I'm working on this type of suggestions having chatgpt and much easier setup.

https://github.com/LiveHelperChat/lhc-chatbot
